Remotion/ObjectBinding/Web/Doc/include/UI/Controls/BocReferenceValue.xml

```diff
@@ -44,10 +44,10 @@
         <item>
           <term>WAI WCAG 1.0</term>
```

Comment:
We can remove all of the WCAG 1.0 stuff from the docs, which should clear up the last two unresolved issues